raiden-network/light-client
# Feature/dapp add connection manager

Fixes #2630

**Short description**
This finally adds the connection manager. For the moment it supports only the WalletConnect provider. More will be added in follow up PRs. This enables the user to use WalletConnect on our public deployment and in a mobile app as the user can configure the required linking options.

**Definition of Done**


**Steps to manually test the change (dApp)**

1. Run the dApp
2. Connect via WalletConnect
3. Try different options
4. Make sure that when you connect the next time, the inserted options are available and you can connect instantly
